app/models/query_record.rb

```diff
@@ -15,12 +19,28 @@ def self.cleanup
                   (@@last_cleanup < 5.minutes.ago) ||
                   ::Rails.env.test?
 
-    connection.delete(%(
-      DELETE FROM #{table_name}
-      WHERE
-        access_count = 0 AND updated_at < DATE_SUB(NOW(), INTERVAL 6 HOUR) OR
-        access_count > 0 AND updated_at < DATE_SUB(NOW(), INTERVAL 1 DAY)
-    ))
+    delete_manager = arel_delete_cleanup(table_name)
+    # Nimmo note: Reviewers can examine the generated SQL like so:
+    # puts(delete_manager.to_sql)
+    connection.delete(delete_manager.to_sql)
+
```

Comment:
What would you and Joe think about simplifying the condition -- ignoring access_count and letting everything live for a day?  Would that let you do this in AR instead of Arel?  I never remember the syntax of date comparisons in ActiveRecord.

Reply:
See below. I like it. Will attempt and commit.

Reply:
Oops - I don't understand how we can do this in AR, considering we are passing the table name as a variable.

Reply:
I did however simplify the conditional.